src/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1337,16 +1423,70 @@ int computeIndexingBin() {
         return GenomicIndexUtil.reg2bin(alignmentStart, alignmentEnd);
     }
 
+    /**
+     *
+     * @return the SAMFileHeader for this record. If the header is null, the following SAMRecord methods may throw
+     * IllegalArgumentExceptions:
```

Comment:
`IllegalArgumentExceptions" -> "exceptions" (since I asked you to change several exception classes above)